data/main.ts

```diff
@@ -98,6 +98,9 @@ const setupInfrastructure = async (): Promise<void> => {
         "ngrok_auth_token",
         "samepage_development_token",
         "vellum_api_key",
+        "langchain_tracing_v2",
+        "langchain_api_key",
+        "langchain_project",
```

Comment:
openai_api_token
google

langchain_tracing_v2 and langchain_project are not sensitive, so we could either:
- inline in the code directly
- hardcode in `app.yaml`